robocop/checkers/tags.py

```diff
@@ -37,5 +37,5 @@ def check_tags(self, node):
     def check_tag(self, tag, node):
         if ' ' in tag.value:
             self.report("tag-with-space", node=node, lineno=tag.lineno, col=tag.col_offset + 1)
-        if 'or' in tag.value.lower() or 'and' in tag.value.lower():
+        if 'OR' in tag.value or 'AND' in tag.value:
```

Comment:
Same as in previous PR 😄 : 
```python
if any(x in ['OR', 'AND'] for x in tag.value):
```

Reply:
In this case tag.value is string (check_tag is invoked on each tag found). 
But using your approach we can do:
```
if any(reserved in tag.value for reserved in ('OR', 'AND'):
```
Should I change it?

Reply:
You're right - it's str, I didn't notice.
If we are here to play than another option is to use:
`any(map(tab.value.__contains__, ['OR', 'AND'))`
But I think that the original one is simple and more understandable.

Reply:
It's starting to look like code golf :D